agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -654,6 +658,45 @@ func (dg *dockerGoClient) inspectContainer(ctx context.Context, dockerID string)
 	return &containerData, err
 }
 
+func (dg *dockerGoClient) TopContainer(ctx context.Context, dockerID string, timeout time.Duration) (*dockercontainer.ContainerTopOKBody, error) {
+	type topResponse struct {
+		top *dockercontainer.ContainerTopOKBody
+		err error
+	}
+	ctx, cancel := context.WithTimeout(ctx, timeout)
+	defer cancel()
+	defer metrics.MetricsEngineGlobal.RecordDockerMetric("CONTAINER_TOP")()
```

Comment:
can we change this from CONTAINER_TOP to TOP_CONTAINER?

Reply:
```
[amazon-ecs-agent]$ grep -r INSPECT_CONTAINER .
Binary file ./out/amazon-ecs-agent matches
./agent/dockerclient/dockerapi/docker_client.go:        defer metrics.MetricsEngineGlobal.RecordDockerMetric("INSPECT_CONTAINER")()
```
CONTAINER_INSPECT -> INSPECT_CONTAINER
which means
CONTAINER_TOP -> TOP_CONTAINER

yes we can.  I'll make that change.